coderd/database/migrations/000219_notifications.up.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TYPE notification_message_status AS ENUM (
```

Comment:
CREATE IF NOT EXISTS?

Reply:
I'd argue that this _should_ fail if the type already exists.

Reply:
Yeah, we are not consistent there if you look at previous migrations.